stable/namerd/templates/config.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}-config
+data:
+  config.yaml: |-
+    admin:
+      ip: 127.0.0.1
+      port: 9991
```

Comment:
Is this the admin port of namerd or linkerd? I'll update the IP to point to the right piece of infrastructure.